airflow/hooks/base_hook.py

```diff
@@ -12,6 +12,13 @@
 from airflow.models import Connection
 from airflow.exceptions import AirflowException
 
+import airflow.configuration as conf
+
+import json
+import requests
+
+base_url = "http://localhost:8080/api/v1/"
```

Comment:
@r39132 sure, this really is experimental, ie. trying out new stuff quick 'n dirty. Nothing you want to merge, but maybe make spinoffs in separate PRs (as with the ZIP functionality)